src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -1,24 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 
-import ProfileStatusWithHooks from './ProfileStatusWithHooks';
+import ProfileStatus from './ProfileStatus';
+import ProfileData from './ProfileData/ProfileData';
+import ProfileDataForm from './ProfileData/ProfileDataForm';
 
 import Loader from '../../../utils/loader';
```

Comment:
The import grouping for this file is off.

Reply:
/fxd